### PR TITLE
Update dependencies / update to the latest `libappindicator-sys`

### PR DESCRIPTION
**Pull request by application-developer-DA**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -13,8 +13,8 @@ keywords = ["linux", "gui", "gtk"]
 travis-ci = { repository = "qdot/libappindicator-rs" }
 
 [dependencies]
-log="0.3"
-glib="0.3"
-gtk="0.2"
-gtk-sys="0.4"
-libappindicator-sys="0.1"
+log = "0.4"
+glib = "0.4"
```
Comment (mardiros) [inline Cargo.toml]: glib = "0.5" now

File: Cargo.toml [hunk 2]
```diff
@@ -13,8 +13,8 @@ keywords = ["linux", "gui", "gtk"]
...
+gtk = "0.3"
```
Comment (mardiros) [inline Cargo.toml]: gtk = "0.4"

File: Cargo.toml [hunk 3]
```diff
@@ -13,8 +13,8 @@ keywords = ["linux", "gui", "gtk"]
...
+gtk-sys = "0.5"
```
Comment (mardiros) [inline Cargo.toml]: gtk-sys="0.6"

**Comment by qdot** [comment]
I'll bring this in then rev the deps again.

**Pull request merged**